qutebrowser/config/configdata.yml

```diff
@@ -1171,6 +1171,27 @@ statusbar.position:
   default: bottom
   desc: Position of the status bar.
 
+statusbar.widgets:
+  default: '{url} [{scroll}] [{history}] [{tabs}]'
+  type:
+    name: FormatString
+    fields:
+      - url
+      - scroll
+      - scroll_raw
+      - history
+      - tabs
+    none_ok: true
+  desc: |
+    Format to use for the statusbar widgets
+    The following placeholders are defined:
+
+    * `{url}`: Current page URL.
+    * `{scroll}`: Percentage of the current page position like `10%`.
+    * `{scroll_raw}`: Raw percentage of the current page position like `10`.
+    * `{history}`: Display an arrow when possible to go back/forward in history.
```

Comment:
(another) nitpick: This line is *one* character too long (see https://github.com/qutebrowser/qutebrowser/commit/99ad1547bc27bd99ef588fed9f900f19f00f41a1). :wink:

Reply:
Please go on. I'm on NixOS and tox doesn't really work so I don't know how to run the tests yet.